server/test/services/cloud/PublicFileNameFormatterTest.java

```diff
@@ -0,0 +1,34 @@
+package services.cloud;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+public class PublicFileNameFormatterTest {
+  @Test
+  public void formatPublicProgramImageFilename_isCorrectlyFormatted() {
+    String filename = PublicFileNameFormatter.formatPublicProgramImageFilename(156);
+
+    assertThat(filename).isEqualTo("program-summary-image/program-156/${filename}");
+  }
+
+  @Test
+  public void isFileKeyForPublicProgramImage_isCorrect_true() {
+    String filename = PublicFileNameFormatter.formatPublicProgramImageFilename(156);
+
+    assertThat(PublicFileNameFormatter.isFileKeyForPublicProgramImage(filename)).isTrue();
+  }
+
+  @Test
+  public void isFileKeyForPublicProgramImage_isApplicantFile_false() {
```

Comment:
nitpick (non-blocking): As written this test sounds like we're testing if it's an applicant file and getting false back. Change test name to `isFileKeyForPublicProgramImage_applicantFile_false` or `isFileKeyForPublicProgramImage_applicantFile_isFalse`.

Reply:
Agreed, updated.